web/packages/design/src/Divider/Divider.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+
+import Flex from 'design/Flex';
+
+/** Renders an "or" divider decorated with a horizontal line. */
+export const Divider = () => (
```

Comment:
- For the component named `Divider` I would expect it to be just a plain old divider that accept props to render horizontal or vertical line. But this renders a Text Divider (https://www.w3schools.com/howto/howto_css_text_divider.asp). 
- The component added in `design` package should have a story.

Reply:
Removing this, per another comment